### ISSUE TITLE
Fails At Evaluation

### ISSUE THREAD

**Issue by Abdulmomen96** [sohiniroych/U-net_using_TF2#1]
When I finished training, the evaluation at the last step gave not promising results, and the output images are just blank images

**Comment by sohiniroych** [comment]
You may have to change the loss function to suit your needs.

**Comment by alexschultz** [comment]
I have the same issue. I downloaded the sample STARE dateset and ran through the notebook and the evaluation step produces plain white images.
I'm also getting this warning message:
/tmp/ipykernel_4267/2938434014.py:21: UserWarning: ./STARE/test/pred/im0003.png_predict.png is a low contrast image
  io.imsave(os.path.join(save_path, files[i]+'_predict.png'),img)

**Comment by sohiniroych** [comment]
This just means you don’t have enough epochs or data na tech sizes are too
less. Increase epochs and data batch sizes and you should have
better results.
Br
Sohini

**Comment by alexschultz** [comment]
Thanks for the quick reply, I actually ended up going over everything again and before when I hit this issue I was converting the dataset images from *.ppm format to *.png. Something in that conversion must have caused image dimensions to not line up with what the code was expecting. I tried again with the raw ppm images from the STARE dataset and now it seems everything is working as expected.
